streamer/Cargo.toml

```diff
@@ -38,6 +39,7 @@ solana-sdk = { workspace = true }
 solana-transaction-metrics-tracker = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
+x25519-dalek = { workspace = true }
 x509-parser = { workspace = true }
```

Comment:
x25519 might have to go after x509 to fix the failed 'dependencies are sorted' check in CI